Resources/Prototypes/Body/Mechanisms/diona.yml

```diff
@@ -0,0 +1,98 @@
+- type: entity
+  id: BaseDionaOrgan
+  parent: BaseItem
+  abstract: true
+  components:
+  - type: Sprite
+    netsync: false
+    sprite: Mobs/Species/Diona/organs.rsi
+  - type: Mechanism
+  - type: Food
+  - type: Extractable
+    grindableSolutionName: organ
+  - type: SolutionContainerManager
+    solutions:
+      organ:
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 10
+
+- type: entity
+  id: OrganDionaBrain
+  parent: BaseDionaOrgan
+  name: brain
+  description: "The source of incredible, unending intelligence. Honk."
+  components:
+  - type: Sprite
+    state: brain
+  - type: Mechanism
+    size: 1
+    compatibility: Biological
+  - type: Metabolizer
+    maxReagents: 2
+    metabolizerTypes: [Human]
+    groups:
+    - id: Medicine
+    - id: Poison
+    - id: Narcotic
+  - type: Input
+    context: "ghost"
+  - type: InputMover
+  - type: MovementSpeedModifier
+    baseWalkSpeed: 0
+    baseSprintSpeed: 0
+  - type: GhostOnMove
+  - type: Brain
+
+- type: entity
+  id: OrganDionaEyes
+  parent: BaseDionaOrgan
+  name: eyes
+  description: "I see you!"
+  components:
+  - type: Sprite
+    layers:
+      - state: eyeball-l
+      - state: eyeball-r
+  - type: Mechanism
+    size: 1
+    compatibility: Biological
+
+- type: entity
+  id: OrganDionaEars
+  parent: BaseDionaOrgan
+  name: ears
+  description: It's a good idea to not eat random berries.
+  components:
+  - type: Sprite
+    state: ears
+  - type: Mechanism
+    size: 1
+    compatibility: Biological
+
+- type: entity
+  id: OrganDionaStomach
+  parent: BaseDionaOrgan
+  name: stomach
+  description: "Gross. This is hard to stomach."
+  components:
+  - type: Sprite
+    state: stomach
+  - type: Mechanism
+    size: 1
+    compatibility: Biological
+  - type: SolutionContainerManager
+    solutions:
+      stomach:
+        maxVol: 250
+  - type: Stomach
+  - type: Metabolizer
+    maxReagents: 6
+    metabolizerTypes: [ Human ]
```

Comment:
They should probably get a plant metabolism so they have unique interactions with botany reagents